runtime/near-test-contracts/build.rs

```diff
@@ -54,6 +54,7 @@ fn build_contract(dir: &str, args: &[&str], output: &str) -> Result<(), Error> {
         .map_err(|err| format!("failed to copy `{}`: {}", src.display(), err))?;
     println!("cargo:rerun-if-changed=./{}/src/lib.rs", dir);
     println!("cargo:rerun-if-changed=./{}/Cargo.toml", dir);
+    println!("cargo:rerun-if-env-changed=CARGO_FEATURE_TEST_FEATURES");
```

Comment:
Can we move this line to where `TEST_FEATURES` env-var is being read to make this a little bit more local?

Otherwise this is actually pretty good as far as solutions are concerned. LGTM!

Reply:
Will do, thanks! I thought this needs to be added after every build command but if that's not the case then placing it locally makes much more sense.